allennlp/data/token_indexers/pretrained_transformer_indexer.py

```diff
@@ -29,27 +31,52 @@ class PretrainedTransformerIndexer(TokenIndexer):
         We use a somewhat confusing default value of ``tags`` so that we do not add padding or UNK
         tokens to this namespace, which would break on loading because we wouldn't find our default
         OOV token.
+    intra_word_tokenization: ``bool``, optional (default=False)
```

Comment:
This parameter is duplicated in the docstring.

Reply:
Oops typo.